HPXMLtoOpenStudio/resources/waterheater.rb

```diff
@@ -188,50 +189,32 @@ def self.apply_combi(model, runner, loc_space, loc_schedule, water_heating_syste
     alternate_stp_sch.setValue(alt_temp)
     new_heater.setIndirectAlternateSetpointTemperatureSchedule(alternate_stp_sch)
 
-    # Create hx setpoint schedule to specify source side temperature
-    hx_stp_sch = OpenStudio::Model::ScheduleConstant.new(model)
-    hx_stp_sch.setName("#{obj_name_combi} HX Spt")
+    # Create setpoint schedule to specify source side temperature
+    source_stp_sch = OpenStudio::Model::ScheduleConstant.new(model)
+    source_stp_sch.setName("#{obj_name_combi} Source Spt")
     boiler_spt_mngr = model.getSetpointManagerScheduleds.select { |spt_mngr| spt_mngr.setpointNode.get == boiler_plant_loop.loopTemperatureSetpointNode }[0]
-    boiler_spt = boiler_spt_mngr.to_SetpointManagerScheduled.get.schedule.to_ScheduleConstant.get.value
-    hx_temp = (UnitConversions.convert(water_heating_system.temperature, 'F', 'C') + deadband(water_heating_system.water_heater_type) / 2.0 + boiler_spt) / 2.0 # tank source side inlet temperature, degree C
-    hx_stp_sch.setValue(hx_temp)
+    boiler_heating_spt = boiler_spt_mngr.to_SetpointManagerScheduled.get.schedule.to_ScheduleConstant.get.value
+    source_temp = (UnitConversions.convert(water_heating_system.temperature, 'F', 'C') + deadband(water_heating_system.water_heater_type) / 2.0 + boiler_heating_spt) / 2.0 # tank source side inlet temperature, degree C
```

Comment:
@yzhou601 We are averaging the water heater setpoint and the boiler setpoint? Can you explain why we are doing that?

Reply:
This is a good question, this is the tank source water temperature. Ideally it should be equal to boiler setpoint, but in old approach we're using heat exchanger object, and setting that temperature will cause problems. Let me try if it works in the new two boiler approach.

Reply:
Using boiler temperature gave a similar result. Except a higher bias on gallons for combi tankless.
![image](https://user-images.githubusercontent.com/32500275/163472236-c2a28565-3fa3-4898-b083-4736e66ada58.png)
But hourly tank temperature still is not bad.
![image](https://user-images.githubusercontent.com/32500275/163472481-c4ea2f09-4eb5-471a-9d98-cd6cca754c6e.png)